src/Parser/RetributionPaladin/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,98 @@
+    26573, // Consecration
+    SPELLS.BLINDING_LIGHT_TALENT.id,
+    642, // Divine Shield
+    SPELLS.LAY_ON_HANDS.id,
+    SPELLS.BLESSING_OF_FREEDOM.id,
+    1022, // Blessing of Protection
+    853, // Hammer of Justice
+    SPELLS.HAND_OF_RECKONING.id,
+  ];
+
+  recordCastTime(
+    castStartTimestamp,
+    globalCooldown,
+    begincast,
+    cast,
+    spellId
+  ) {
+    super.recordCastTime(
+      castStartTimestamp,
+      globalCooldown,
+      begincast,
+      cast,
+      spellId
+    );
+    this.verifyCast(begincast, cast, globalCooldown);
+  }
+  verifyCast(begincast, cast, globalCooldown) {
+    if (cast.ability.guid !== SPELLS.FLASH_OF_LIGHT.id) {
+      return;
+    }
+    const castTime = cast.timestamp - begincast.timestamp;
+    if (!this.constructor.inRange(castTime, globalCooldown, 50)) { // cast times seem to fluctuate by 50ms, not sure if it depends on player latency, in that case it could be a lot more flexible
+      console.warn(`Expected Flash of Light cast time (${castTime}) to match GCD (${Math.round(globalCooldown)}) @${formatDuration((cast.timestamp - this.owner.fight.start_time) / 1000)}`, this.combatants.selected.activeBuffs());
+    }
+  }
+
+  countsAsHealingAbility(cast) {
+    const spellId = cast.ability.guid;
+    if (spellId === SPELLS.HOLY_SHOCK_CAST.id && !cast.targetIsFriendly) {
+      debug && console.log(`%cABC: ${cast.ability.name} (${spellId}) skipped for healing time; target is not friendly`, 'color: orange');
+      return false;
+    }
+    return super.countsAsHealingAbility(cast);
+  }
+
+  static inRange(num1, goal, buffer) {
+    return num1 > (goal - buffer) && num1 < (goal + buffer);
+  }
+
+  suggestions(when) {
+    const nonHealingTimePercentage = this.totalHealingTimeWasted / this.owner.fightDuration;
+    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+
+    when(nonHealingTimePercentage).isGreaterThan(0.3)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('Your non healing time can be improved. Try to reduce the delay between casting spells and try to continue healing when you have to move.')
+          .icon('petbattle_health-down')
+          .actual(`${formatPercentage(actual)}% non healing time`)
+          .recommended(`<${formatPercentage(recommended)}% is recommended`)
+          .regular(recommended + 0.1).major(recommended + 0.15);
+      });
+    when(deadTimePercentage).isGreaterThan(0.2)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('Your dead GCD time can be improved. Try to Always Be Casting (ABC); try to reduce the delay between casting spells and when you\'re not healing try to contribute some damage.')
+          .icon('spell_mage_altertime')
+          .actual(`${formatPercentage(actual)}% dead GCD time`)
+          .recommended(`<${formatPercentage(recommended)}% is recommended`)
+          .regular(recommended + 0.15).major(1);
+      });
+  }
+  statistic() {
```

Comment:
This should include a statistic for displaying the dead GCD time, for example:

https://github.com/MartijnHols/WoWAnalyzer/blob/4fcfe5839acc7d929944e57bee5652e1ede58f79/src/Parser/AfflictionWarlock/Modules/Features/AlwaysBeCasting.js#L63-L74